project/plugins.sbt

```diff
@@ -1,15 +1,12 @@
-resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
```

Comment:
Would you mind explaining why this resolver is no longer required? (Just for my own understanding!)